src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -8,432 +8,56 @@ import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+		}else{
+			content = <UploadDetachedFileMeta setSubmissionState={this.props.setSubmissionState} setSubmissionId={this.props.setSubmissionId.bind(this)} 
+				history={this.props.history} submission={this.props.submission} validate={this.validate.bind(this)}></UploadDetachedFileMeta>;
 		}
 
-		let errorMessage = null;
-		if (this.state.detachedAward.error.show) {
-			errorMessage = <div className="alert alert-error text-left" role="alert">
-								<span className="usa-da-icon error-icon"><Icons.ExclamationCircle /></span>
-								<div className="alert-header-text">{this.state.detachedAward.error.header}</div>
-								<p>{this.state.detachedAward.error.description}</p>
-							</div>;
-		}
-		
 		return (
 			<div className="usa-da-upload-detached-files-page">
 				<div className="usa-da-site_wrap">
-					<div className="usa-da-page-content">
-						<Navbar activeTab="submissionGuide" />
-						<div className="usa-da-content-dark">
-							<div className="container">
-								<div className="row usa-da-page-title">
-									<div className="col-lg-12 mt-40 mb-20">
-										<div className="display-2">
-											Upload Bi-Monthly Financial Assistance Data
-										</div>
-									</div>
-								</div>
-							</div>
-						</div>
-
-						<div className="container center-block">
-							<div className="row text-center usa-da-select-agency">
-								<div className="col-lg-offset-2 col-lg-8 mt-60 mb-60">
-									<h5>Please begin by telling us about files you would like to upload</h5>
-									<div className="select-agency-holder">
-										<div className="row usa-da-select-agency-label">
-											The files will be used when submitting data for...
-										</div>
-
-										<div className="row">
-											<div className="col-sm-12 col-md-12 typeahead-holder" data-testid="agencytypeahead">
-												<SubTierAgencyListContainer placeholder="Enter the name of the reporting sub-tier agency" onSelect={this.handleChange.bind(this)} customClass={subTierAgencyClass} internalValue='agency_code' disabled={this.state.showValidationBox} />
-												<div className={"usa-da-icon usa-da-form-icon" + subTierAgencyClass}>
-													{subTierAgencyIcon}
-												</div>
-											</div>
-										</div>
-
-										<ReactCSSTransitionGroup transitionName="usa-da-meta-fade" transitionEnterTimeout={600} transitionLeaveTimeout={200}>
-											{datePicker}
-										</ReactCSSTransitionGroup>
-
-										<ReactCSSTransitionGroup transitionName="usa-da-meta-fade" transitionEnterTimeout={600} transitionLeaveTimeout={200}>
-											{uploadFilesBox}
-										</ReactCSSTransitionGroup>
-
-										<ReactCSSTransitionGroup transitionName="usa-da-meta-fade" transitionEnterTimeout={600} transitionLeaveTimeout={200}>
-											{validationBox}
-										</ReactCSSTransitionGroup>
-
-										{errorMessage}
-									</div>
-								</div>
-							</div>
-						</div>
-					</div>
+				{content}
```

Comment:
Usually not a big deal, but since I made an actual comment already might as well fix this while you're in there. Indent this one more level.